spec/test_app/app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def root
```

Comment:
I'm not attached to `root` -- I was just following the example in the bootstrap gem. It connects with the `root` route directive so it made a certain sense to me.

The Rails REST conventions don't seem to be followed in the static page examples:
https://www.railstutorial.org/book/static_pages#_code-static_pages_controller

My expectation was that we would just expand this single page to cover the various test scenarios. Happy to change if you think `index` is more intuitive.